support.sch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<schema xmlns="http://purl.oclc.org/dsdl/schematron" queryBinding="xslt">
+
+    <pattern id="attribute-whitelist">
+        <rule context="@*">
+            <assert test="
+                name()='xml:lang'
+            ">
+                @<name/> is ignored.
+            </assert>
+        </rule>
+    </pattern>
+
+    <pattern id="element-whitelist">
+        <rule context="*">
+            <assert test="
+                name()='article'
+            ">
+                &lt;<name/>&gt; is ignored.
```

Comment:
what would be an example of an element that is not `ignored` (not implemented and hence won't be picked up by anything in `browser`) but should actually be forbidden? e.g. a variant for `article-title` that we want to avoid in order to favor other alternatives

Reply:
I don't think we ever _forbid_ any JATS (this isn't validation); if you want to do something that we don't support then you can use configuration or code to add support for it, or you change your XML to match what is supported.

Reply:
Might be a few things we mandate though, like there must be a `<title-group>` in the `<article-meta>`.